libraries/SITL/SIM_Frame.cpp

```diff
@@ -342,7 +345,8 @@ void Frame::load_frame_params(const char *model_json)
     if (fd == -1) {
         AP_HAL::panic("%s failed to load\n", model_json);
     }
-    char buf[st.st_size];
+    char buf[st.st_size+1];
```

Comment:
```suggestion
    char buf[st.st_size+1] = {0}; 
```
maybe ?

Reply:
clang++ was unhappy with the attempt to initialise this.

The memset will work :-)

Reply:
We use the brackets everywhere else in the code.. why not be consistent?